README.md

```diff
@@ -155,6 +155,12 @@ This call BeePlugin `toggleStructure` which toggle the structure helper on the e
 ### toggleComments()
 This call BeePlugin `toggleStructure` which toggle the comments section on the editor's sidebar.
 
+### togglePreview()
+This call BeePlugin `togglePreview` which open/close the message preview behavior within the editor.
+
+### showComment(uuid)
```

Comment:
... which opens a specific comment and scrolls the stage to the related element. The method needs one parameter, a comment uuid as a string.